<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:repeat, x1:0.531, y1:1, x2:0.525584, y2:0, stop:0 rgba(0, 180, 255, 255), stop:0.482955 rgba(70, 92, 124, 255), stop:1 rgba(0, 0, 0, 246));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_help">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>390</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,200);    
color: rgb(0, 0, 0); 
font: 12pt;
font-family: &quot;Microsoft Sans Serif&quot;;
font-weight: 400;
border-radius: 7px;
}
QPushButton:hover{ 
background-color:rgba(255,255,255,255);
}</string>
    </property>
    <property name="text">
     <string>Ajuda</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_quit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>420</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Sans Serif</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,200);    
color: rgb(0, 0, 0); 
font: 12pt;
font-family: &quot;Microsoft Sans Serif&quot;;
font-weight: 400;
border-radius: 7px;
}
QPushButton:hover{ 
background-color:rgba(255,255,255,255);
}</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_menu">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>360</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,200);    
color: rgb(0, 0, 0); 
font: 12pt;
font-family: &quot;Microsoft Sans Serif&quot;;
font-weight: 400;
border-radius: 7px;
}
QPushButton:hover{ 
background-color:rgba(255,255,255,255);
}</string>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_company">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Sans Serif</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,200);    
color: rgb(0, 0, 0); 
font: 12pt;
font-family: &quot;Microsoft Sans Serif&quot;;
font-weight: 400;
border-radius: 7px;
}
QPushButton:hover{ 
background-color:rgba(255,255,255,255);
}</string>
    </property>
    <property name="text">
     <string>Adicionar nova empresa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_train_AI">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Sans Serif</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,200);    
color: rgb(0, 0, 0); 
font: 12pt;
font-family: &quot;Microsoft Sans Serif&quot;;
font-weight: 400;
border-radius: 7px;
}
QPushButton:hover{ 
background-color:rgba(255,255,255,255);
}</string>
    </property>
    <property name="text">
     <string>Treinar modelo I.A.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_restart_db">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Sans Serif</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,200);    
color: rgb(0, 0, 0); 
font: 12pt;
font-family: &quot;Microsoft Sans Serif&quot;;
font-weight: 400;
border-radius: 7px;
}
QPushButton:hover{ 
background-color:rgba(255,255,255,255);
}</string>
    </property>
    <property name="text">
     <string>Reiniciar banco de dados</string>
    </property>
   </widget>
   <widget class="QLabel" name="window_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:1 rgba(0, 0, 0, 0));
color: rgb(240, 255, 255);
font-size: 36,</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Configurações&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_goback">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,200);    
color: rgb(0, 0, 0); 
font: 12pt;
font-family: &quot;Microsoft Sans Serif&quot;;
font-weight: 400;
border-radius: 7px;
}
QPushButton:hover{ 
background-color:rgba(255,255,255,255);
}</string>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_edit_db">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Sans Serif</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,200);    
color: rgb(0, 0, 0); 
font: 12pt;
font-family: &quot;Microsoft Sans Serif&quot;;
font-weight: 400;
border-radius: 7px;
}
QPushButton:hover{ 
background-color:rgba(255,255,255,255);
}</string>
    </property>
    <property name="text">
     <string>Editar banco de dados</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
